### PR TITLE
Pre-JIP-20/21 smol server change to get posts from the db

### PR DESCRIPTION
**Pull request by srhsrhsrhsrh**
Added the getPosts endpoint so that I can get real posts for geospatial querying and text moderation

image

### CODE REVIEW

File: packages/server/src/data/db/db-operations.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const Post = require("../models/post-model");
```
Comment (dryu99) [inline packages/server/src/data/db/db-operations.js:1]: are we gonna dump all our db calls here or is this just a temporary thing? ow mayhaps we could make a db-operations module for each collection in our db (e.g. posts, users, etc)

File: packages/server/src/app.js [hunk 2]
```diff
@@ -29,6 +32,9 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, "public")));
 
 app.use("/", indexRouter);
+app.use("/api/posts", indexRouter);
```
Comment (dryu99) [inline packages/server/src/app.js:35]: mayhaps we could implement a router for each resource

File: packages/server/src/data/db/db-operations.js (hunk 1 above)
Comment (srhsrhsrhsrh) [reply packages/server/src/data/db/db-operations.js:1]: mayhaps I'll make db-operations a folder for later

File: packages/server/src/app.js (hunk 2 above)
Comment (srhsrhsrhsrh) [reply packages/server/src/app.js]: look now mustache man

File: packages/server/src/app.js [hunk 3]
```diff
@@ -19,8 +20,11 @@ liveReloadServer.server.once("connection", () => {
   }, 100);
 });
 
+const db = connectToDb().connection;
```
Comment (jpfranci) [inline packages/server/src/app.js:23]: This actually returns a promise

**Review by jpfranci — APPROVED** [review]

**Pull request merged**